src/dini/headers.js

```diff
@@ -0,0 +1,205 @@
+//          - url: URI for the person's home page
+//          - company: the person's company
+//          - companyURL: the URI for the person's company
+//          - mailto: the person's email
+//          - note: a note on the person (e.g. former editor)
+//  - authors: an array of people who are contributing authors of the document.
+//  - formerEditors: an array of people that had earlier edited the document but no longer edit.
+//  - subtitle: a subtitle for the specification
+//  - publishDate: the date to use for the publication, default to document.lastModified, and
+//      failing that to now. The format is YYYY-MM-DD or a Date object.
+//  - alternateFormats: a list of alternate formats for the document, each of which being
+//      defined by:
+//          - uri: the URI to the alternate
+//          - label: a label for the alternate
+//          - lang: optional language
+//          - type: optional MIME type
+//  - logos: a list of logos to use instead of the W3C logo, each of which being defined by:
+//          - src: the URI to the logo (target of <img src=>)
+//          - alt: alternate text for the image (<img alt=>), defaults to "Logo" or "Logo 1", "Logo 2", ...
+//            if src is not specified, this is the text of the "logo"
+//          - height: optional height of the logo (<img height=>)
+//          - width: optional width of the logo (<img width=>)
+//          - url: the URI to the organization represented by the logo (target of <a href=>)
+//          - id: optional id for the logo, permits custom CSS (wraps logo in <span id=>)
+//          - each logo element must specify either src or alt
+//  - otherLinks: an array of other links that you might want in the header (e.g., link github, twitter, etc).
+//         Example of usage: [{key: "foo", href:"https://b"}, {key: "bar", href:"https://"}].
+//         Allowed values are:
+//          - key: the key for the <dt> (e.g., "Bug Tracker"). Required.
+//          - value: The value that will appear in the <dd> (e.g., "GitHub"). Optional.
+//          - href: a URL for the value (e.g., "https://foo.com/issues"). Optional.
+//          - class: a string representing CSS classes. Optional.
+//  - license: can be one of the following
+//      - "cc-by-sa"
+//      - "cc-by"
+//      - "cc0"
+import { ISODate, htmlJoinAnd } from "../core/utils.js";
+import headersTmpl from "../w3c/templates/headers.js";
+import { hyperHTML } from "../core/import-maps.js";
+import { pub } from "../core/pubsubhub.js";
+
+export const name = "dini/headers";
+
+const W3CDate = new Intl.DateTimeFormat(["de-DE"], {
+  timeZone: "UTC",
+  year: "numeric",
+  month: "long",
+  day: "2-digit",
+});
+
+const status2text = {
+  unofficial: "Entwurf vom",
+  base: "Spezifikation vom",
+};
+
+/**
+ * @param {*} conf
+ * @param {string} prop
+ * @param {string | number | Date} fallbackDate
+ */
+function validateDateAndRecover(conf, prop, fallbackDate = new Date()) {
```

Comment:
We should probably export this from core/headers.js, then we avoid the duplicate code.